tiflash/troubleshoot-tiflash.md

```diff
@@ -97,3 +98,168 @@ Take the following steps to handle the data file corruption:
 1. Refer to [Take a TiFlash node down](/scale-tidb-using-tiup.md#scale-in-a-tiflash-cluster) to take the corresponding TiFlash node down.
 2. Delete the related data of the TiFlash node.
 3. Redeploy the TiFlash node in the cluster.
+
+## TiFlash analysis is slow
+
+If a statement contains operators or functions not supported in MPP mode, TiDB does not select MPP mode. This is because the analysis of the statement is slow. In this case, you can execute the `EXPLAIN` statement to check for operators or functions not supported in MPP mode.
+
+{{< copyable "sql" >}}
+
+```sql
+create table t(a datetime);
+alter table t set tiflash replica 1;
+insert into t values('2022-01-13');
+set @@session.tidb_enforce_mpp=1;
+explain select count(*) from t where subtime(a, '12:00:00') > '2022-01-01' group by a;
+show warnings;
+```
+
+In this example, the warning message shows that TiDB does not select MPP mode because TiDB 5.4 and earlier versions do not support the `subtime` function.
```

Comment:
```suggestion
In this example, the warning message shows that TiDB does not select the MPP mode because TiDB 5.4 and earlier versions do not support the `subtime` function.
```